terraform/api_gateway.tf.json.template.py

```diff
@@ -204,6 +204,40 @@ def for_domain(cls, domain):
                     'rule': [
                         {
                             'priority': 0,
+                            'name': 'LabelPutRequests',
```

Comment:
```suggestion
                            'name': 'LabelExpensiveRequests',
```

but if it is possible to easily change rule names, and to use underscores in them, I'd like to use the snake case version of that, and rename all existing rules using that convention as well.